src/ActionsObservable.js

```diff
@@ -23,6 +25,14 @@ export class ActionsObservable extends Observable {
     return observable;
   }
 
+  catchAndRelease(action) {
+    return this::catch((error, source) => {
```

Comment:
feel free to ignore this for now as we're still discussing the viability in general